src/lib.rs

```diff
@@ -37,7 +40,7 @@ const fn chain_id() -> u64 {
 const fn state_version() -> u32 {
     if is_testnet() {
         // Testnet.
-        3
+        4
```

Comment:
Note to viewers: This is not int -> chain mapping but rather the version of a chain's state. Bumping this to `4` means we are migrating Emerald's testnet state in 11.0.0-testnet release.